eslint.config.mjs

```diff
@@ -0,0 +1,3 @@
+import neolutionEslintConfig from "@neolution-ch/eslint-config-neolution";
+
+export default [...neolutionEslintConfig.configs.flat.typescript, { rules: { "unicorn/no-null": "off" } }];
```

Comment:
here we should add the custom rules that were defined in .eslintrc.cjs
or go through them and discuss whether to add some of them directly to the neolution config, like the double quotes, max-lines, complexity, ...

Reply:
I checked the eslint-config-neolution but there's nothing about the max-lines, complexity, etc... 
The idea, you mentioned, is to keep the eslint-config-neolution with just the recommended rules, so if those rules are not defined in the package they must be defined here, no?

Reply:
sorry that was my fault, I didn't push the newest changes to the eslint package 😒 now its pushed and the mentioned rules are visible.

Reply:
awesome, nice that those rules are now in the package and not in every project anymore.
So what's left here would be to extend the neolution config to enable the "jsdoc/require-jsdoc" since in the neolution package is configured as "off".

The "jsdoc/require-jsdoc" rule is quite hard to configure, but would be nice (in future) to maybe have a generic configuration already in the neolution config to enable it for certain projects, especially public libraries like this one or  the various react packages